static/index.html

```diff
@@ -63,7 +66,7 @@ <h3 id="title"></h3>
             document.getElementById("interlude").style.visibility = "visible";
             document.getElementById("playing").style.visibility = "hidden";
         }
-    
+        console.log(stateURL.href)
```

Comment:
remove this line, i accidentally added this console log

Reply:
done